### PR TITLE
Create `<h3>` headings for profile page. Replace `<label>` which have…

### LINKED ISSUE #36912: [BITV] 9.1.3.1a/6.1 - All label elements of form elements were mistakenly marked as headings. (1)
Since they are not headings, the heading markup should be removed. Only "Profile picture", "Details", and "Profile visibility" should be implemented as "h3" headings without labels inside. 

- [x] Fix
- [x] Backport

0daff04b35b3395981d62d5f87436606

ea791921f8d91989b62585d40226bf0f

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-be75bbae42-v6-n1

</details>

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Create `<h3>` headings for profile page. Replace `<label>` which have no connected `<input>` with `<h3>` heading

* Resolves: https://github.com/nextcloud/server/issues/36912

## Summary

No visual changes.

Screenshot from 2023-03-14 14-10-26

Screenshot from 2023-03-14 14-11-19

Screenshot from 2023-03-14 14-11-55

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by Pytal — APPROVED** [review]

File: apps/settings/css/settings.scss [hunk 1]
```diff
@@ -56,7 +56,7 @@ input {
 	.section {
 		padding: 10px 30px;
 
-		h3 {
+	  .headerbar-label {
```
Comment (ChristophWurst) [inline apps/settings/css/settings.scss:59]: indentation

**Review by ChristophWurst — APPROVED** [review]

File: apps/settings/css/settings.scss (hunk 1 above)
Comment (JuliaKirschenheuter) [reply apps/settings/css/settings.scss:59]: what do you mean i should change?

Comment (ChristophWurst) [reply apps/settings/css/settings.scss:59]: Bildschirmfoto vom 2023-03-17 16-11-24

^ this. the h3 was indented with tabs. the class is indented with a mix of tab and spaces.

Comment (ChristophWurst) [inline apps/settings/css/settings.scss:59]:
```suggestion
		.headerbar-label {
```

**Review by ChristophWurst — APPROVED** [review]

**Comment by juliushaertl** [comment]
Failures unrelated

**Pull request merged**